packages/suite/src/config/wallet/networks.ts

```diff
@@ -329,6 +329,50 @@ const networks = [
         },
         features: ['rbf', 'sign-verify'],
     },
+    {
+        name: 'Bitcoin Regtest',
+        networkType: 'bitcoin',
+        symbol: 'regtest',
+        bip43Path: "m/84'/1'/i'",
+        decimals: 8,
+        testnet: true,
+        label: 'TR_TESTNET_COINS_LABEL',
+        explorer: {
+            tx: 'http://localhost:19121/tx/',
+            account: 'http://localhost:19121/xpub/',
+        },
+        features: ['rbf', 'sign-verify'],
+    },
```

Comment:
missing Regtest taproot

Reply:
It's true, we have taproot now. I added it with 9d9f191890b0168ee4c076761a0bd11a3c12cd0b taken inspiration from bitcoin testnet.